### PR TITLE
[Fix] fix addfile cannot parse

### PR DESCRIPTION
**Pull request by Zzm0809**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java [hunk 1]
```diff
@@ -276,13 +276,13 @@ public static Optional<JobClient> executeJarJob(String type, Executor executor,
             if (Operations.getOperationType(sqlStatement) == SqlType.ADD) {
                 File[] info = AddJarSqlParseStrategy.getInfo(sqlStatement);
                 Arrays.stream(info).forEach(executor.getDinkyClassLoader().getUdfPathContextHolder()::addOtherPlugins);
-                if ("kubernetes-application".equals(type)) {
+                if (GatewayType.KUBERNETES_APPLICATION.getLongValue().equals(type)) {
```
Comment (gaoyan1998) [inline dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java]: should use `GatewayType.isKubernetsApplicationMode()`

File: dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java [hunk 2]
```diff
@@ -276,13 +276,13 @@ public static Optional<JobClient> executeJarJob(String type, Executor executor,
...
                     executor.addJar(info);
                 }
             } else if (Operations.getOperationType(sqlStatement) == SqlType.ADD_FILE) {
                 File[] info = AddFileSqlParseStrategy.getInfo(sqlStatement);
                 Arrays.stream(info).forEach(executor.getDinkyClassLoader().getUdfPathContextHolder()::addFile);
-                if ("kubernetes-application".equals(type)) {
+                if (GatewayType.KUBERNETES_APPLICATION.getLongValue().equals(type)) {
```
Comment (gaoyan1998) [inline dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java]: should use GatewayType.isKubernetsApplicationMode()

**Review by gaoyan1998 — CHANGES_REQUESTED** [review]
should use `GatewayType.isKubernetsApplicationMode()`

**Comment by Zzm0809** [comment]
@gaoyan1998 PTAL

**Review by gaoyan1998 — APPROVED** [review]
LGTM

**Pull request merged**